edx_repo_tools/find_dependencies/find_dependencies.py

```diff
@@ -119,6 +119,8 @@ def find_real_url(url: str) -> Optional[str]:
             # I didn't know you could get 429 from https://github.com, but you can...
             wait = int(resp.headers.get("Retry-After", 10))
             time.sleep(wait + 1)
+        elif resp.status_code in {301, 302}:
```

Comment:
I think instead of doing this manually you should let `requests` handle it.

https://docs.python-requests.org/en/latest/user/quickstart/#redirection-and-history

This way you follow all the redirects in case things move multiple times.

Reply:
Yes, I'm just realizing this, and wondering why the default of allow_redirects=True isn't applying here.

Reply:
oh, because I used HEAD.

Reply:
Fixed.